.github/workflows/deploy.yml

```diff
@@ -150,6 +180,38 @@ jobs:
         run: |
           kubectl --namespace=${{ inputs.namespace }} --kubeconfig $(pwd)/kubeconfig apply -f ./keycloak-secret-dev.yaml
 
+      - name: Create Database for Keycloak
+        run: |
+          # Generate timestamp for job name uniqueness
+          DATE=$(date +%Y%m%d%H%M%S)
+
+          # Define job name for Keycloak database creation job
+          HELM_KEYCLOAK_JOB_NAME="psql-db-${DATE}-keycloak"
```

Comment:
would not is be nice to also have the db name in the name, or at least the ticket number?

Reply:
and these helm charts are deployed in the default namespace, could the psql-config get used or the namespace of the Db itself? Then the name in the job would not be necessary anymore

Reply:
The job will already be deployed in the psql-config namespace, we do not want the job (& Secret) in the namespace of the application. I will append the ticket-nr to the job name. I think the whole namespace would be to much.

Reply:
but the helm chart itself seems to be in the default namespace